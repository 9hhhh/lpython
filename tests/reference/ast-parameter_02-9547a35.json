{
    "basename": "ast-parameter_02-9547a35",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/parameter_02.f90",
    "infile_hash": "47758a0a5567afea1616d2c88b1ccfcd08a33ae79b4bb8f156f9426a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-parameter_02-9547a35.stdout",
    "stdout_hash": "c0ba8685bd300b47afabb6d1d050537d93f62d5c4050356257e47469",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}