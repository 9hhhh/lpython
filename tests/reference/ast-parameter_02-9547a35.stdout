(TranslationUnit [(Subroutine a_proc [] [] [(Declaration [(i "integer" [] [] [] ())]) (ParameterStatement [(i 1)])] [(Print () [i])] []) (Module a_mod [] [] []) (Program parameter_02 [(Use a_mod [(UseSymbol i ())])] [] [(SubroutineCall a_proc [] []) (Print () [i])] [])])
